package/src/components/MessageInput/FileUploadPreview.tsx

```diff
@@ -104,7 +105,14 @@ const UnsupportedFileTypeOrFileSizeIndicator = ({
       </Text>
     </View>
   ) : (
-    <Text style={[styles.fileSizeText, { color: grey }, fileSizeText]}>
+    <Text
```

Comment:
Are we going to repeat oursleves in every `Text` component with `writingDirection: 'rtl'` if RTL?

Reply:
This was mistakenly commited while I was switching branches...

Reply:
Removed it in my latest commit...